### PR TITLE
add ice plugin to krew index

### PR DESCRIPTION
**Pull request by NimbleArchitect**
local plugin install works:

`
$ kubectl krew install --manifest ./release.yaml
Installing plugin: ice
Installed plugin: ice
\
 | Use this plugin:
 |      kubectl ice
 | Documentation:
 |      https://github.com/NimbleArchitect/kubectl-ice
 | Caveats:
 | \
 |  | Usage:
 |  |   $ kubectl ice
 |  | 
 |  | For additional options:
 |  |   $ kubectl ice --help
 |  |   or https://github.com/NimbleArchitect/kubectl-ice/blob/v0.0.2/README.md
 | /
/
`

### CODE REVIEW

**Comment by ahmetb** [comment]
:robot: _Beep beep! I’m a robot speaking on behalf of @ahmetb._ :robot:

-----

Thanks for submitting your kubectl plugin to Krew!
One of the krew-index maintainers will review it soon. Note that the reviews for new plugin submissions may take a few days.

In the meanwhile, here are a few tips to make your plugin manifest better:
* Make sure your plugin follows the [best practices](https://krew.sigs.k8s.io/docs/developer-guide/develop/best-practices/).
* Eliminate redundant wording form `shortDescription` (it should be max 50 characters).
* Try to word wrap your `description` to 80-character lines (no usage examples, please).

Thanks for your patience!
/kind new-plugin

File: plugins/ice.yaml
Comment (ahmetb) [inline plugins/ice.yaml:50]: Delete up until view and capitalise as View

File: plugins/ice.yaml
Comment (ahmetb) [inline plugins/ice.yaml:53]: Please delete this section. We already print how to use the plugin.

**Review by ahmetb — CHANGES_REQUESTED** [review]
Can you provide example output? Thanks.

**Comment by NimbleArchitect** [comment]
example output of all commands is available in the [readme](https://github.com/NimbleArchitect/kubectl-ice/blob/main/README.md)

I have added a few example outputs below or is the output supposed to go in the ice.yaml somewhere?

```
$ kubectl ice command mypod
T  CONTAINER     COMMAND    ARGUMENTS
S  app-watcher   -          -
S  app-broken    /bin/bash  -s exit 1
S  myapp         -          -
I  app-init      init.sh    -
```
```
$ kubectl ice cpu mypod
T  CONTAINER    USED  REQUEST  LIMIT   %REQ  %LIMIT
S  app-watcher  0     20m      50m     0     0
S  app-broken   0     20m      50m     0     0
S  myapp        1     500m     1       200   100
I  app-init     0        0     0       0     0
```
```
$ kubectl ice volumes mypod
CONTAINER    VOLUME                 TYPE      BACKING SIZE  RO     MOUNT-POINT                                    
app-init     kube-api-access-k7hvs  Projected               true   /var/run/secrets/kubernetes.io/serviceaccount  
app-watcher  appsafe                EmptyDir  Memory        false  /mnt/appsafe                                   
app-watcher  work                   EmptyDir  Memory        false  /mnt/work                                      
app-watcher  shareme                EmptyDir  Memory        false  /etc/shareme                                   
app-watcher  kube-api-access-k7hvs  Projected               true   /var/run/secrets/kubernetes.io/serviceaccount  
app-broken   work                   EmptyDir  Memory        false  /mnt/work                                      
app-broken   appsafe                EmptyDir  Memory        true   /mnt/appsafe                                   
app-broken   kube-api-access-k7hvs  Projected               true   /var/run/secrets/kubernetes.io/serviceaccount  
myapp        appsafe                EmptyDir  Memory        true   /mnt/appsafe                                   
myapp        work                   EmptyDir  Memory        true   /mnt/work                                      
myapp        kube-api-access-k7hvs  Projected               true   /var/run/secrets/kubernetes.io/serviceaccount 
```
```
$ kubectl ice status mypod
T  CONTAINER    READY STARTED  RESTARTS  STATE       REASON     EXIT-CODE  SIGNAL  TIMESTAMP                      MESSAGE  
S  app-broken   true  true     0         Running                                   2022-02-28 11:04:24 +0000 GMT           
S  app-watcher  true  true     0         Running                                   2022-02-28 11:04:24 +0000 GMT           
S  myapp        true  true     0         Running                                   2022-02-28 11:04:26 +0000 GMT           
I  app-init     true           0         Terminated  Completed  0          0       2022-02-28 11:04:17 +0000 GMT           
```

**Comment by ahmetb** [comment]
I don't understand the name "ice". but it looks good.
If you have a better name in mind, there's still time to change, as it seems to be a new plugin.
/lgtm
/approve
/hold

Also, after it's merged, please consider setting up [Krew release automation](https://krew.sigs.k8s.io/docs/developer-guide/release/automating-updates/) which helps you skip manually making updates to your Krew manifests on each new version and send a pull request. It’s a GitHub workflow bot that can run every time you push a new tag. Those PRs are automatically approved.

**Comment by NimbleArchitect** [comment]
Thanks for the info and heads up I'll look into the automation.

I did call it "pod" during dev/testing but changed it before sending the pull request as I didn't want to confuse newbies, ended up with ice as it was easy to type and remember but I'm open to ideas if you have any ;)

**Comment by ahmetb** [comment]
Ack, I just am just not sure what "ice" means here. Since it's a unique name, I won't pose any objections to it but it might impact your discoverability. (Names like "container" or "pod" would not be acceptable in krew-index as they're too broad.) I like the plugin, it's unique in what it does (listing items over multiple containers on a Pod). 

/hold cancel

**Comment by NimbleArchitect** [comment]
Thank you, it can also list over multiple pods.

Do I need to do something now or is it a case of just waiting (sorry, i'm new to pull requests)

**Pull request merged**
